src/Microsoft.FeatureManagement/Telemetry/EvaluationEvent.cs

```diff
@@ -13,6 +15,11 @@ public class EvaluationEvent
         /// </summary>
         public FeatureDefinition FeatureDefinition { get; set; }
 
+        /// <summary>
+        /// The targeting context used to evaluate the feature.
+        /// </summary>
+        public ITargetingContext TargetingContext { get; set; }
```

Comment:
I'd like to use `TargetingContext` instead of ITargetingContext, similarly to how `GetVariantAsync` takes `TargetingContext` rather than `ITargetingContext`.

https://github.com/microsoft/FeatureManagement-Dotnet/blob/2d7073a2006f478836290cfe4aae7b7bd920bbbd/src/Microsoft.FeatureManagement/IVariantFeatureManager.cs#L54

`ITargetingContext` exists to satisfy ContextualTargetingFilter.

Reply:
Good point, adjusted.